Update documentation about options to switch font size and position of the text (#3989)
openvinotoolkit/cvat: site/content/en/docs/manual/basics/settings.md

  ![](/images/image152_detrac.jpg)

- `Position of a text` - text positioning mode selection:

>   - `Auto` - the object details will be automatically placed where there is free space.

Reviewer: ```suggestion
  - `Auto` - the object details will be automatically placed where free space is.
```

>   - `Center` - the object details will be in the center of the object.

Reviewer: ```suggestion
  - `Center` - the object details will be embedded to a corresponding object if possible.
```
